### DISCUSSION TITLE
when I clean the .cache,some files maybe delete

### DISCUSSION THREAD

**Discussion by Homer-Max** [mylinuxforwork/dotfiles#135] [General]
I used rmlint to clean the .cache, a lot of files were deleted. the power menu has no themes and icons.
so I reinstall ml4w,but it doesn't work

**Comment by Homer-Max** [comment]
edit ~/.config/wlogout/style.css, and delete @import '../../.cache/wal/colors-wlogout.css'; ,
it's ok now.
I think this is a bug

**Reply by mylinuxforwork** [reply]
I will check.
